bbb-exporter/collector.py

```diff
@@ -66,6 +81,15 @@ def collect(self):
         bbb_meetings_video_participants.add_metric([], no_video_participants)
         yield bbb_meetings_video_participants
 
+
+        bbb_meetings_participant_clients = GaugeMetricFamily('bbb_meetings_participants_clients',
```

Comment:
You have an additional _s_ in the metrics name, it should be: `bbb_meetings_participant_clients` like you documented in the README.